Makefile

```diff
@@ -15,11 +15,13 @@ FONT_AWESOME_TARGET        := themes/$(DOCSY_COMMIT_FOLDER)/assets/vendor/$(FONT
 
 DEV_IMAGE_REGISTRY_NAME    := fluxcd
 HUGO_VERSION               ?= 0.78.2
```

Comment:
This is at odds with https://github.com/fluxcd/website/blob/main/package.json#L4 - I think that's how netlify knows to use the extended version.

Reply:
I corrected the Makefile to use 0.83.1 matching package-lock.json